libs/libmbim/Makefile

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libmbim
+PKG_VERSION:=1.18.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.freedesktop.org/software/libmbim
+PKG_HASH=7723f2ca2286c5286ffba7331cb4520099982e55355a7af6f45bdce6b5d39268
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING COPYING.LIB
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/libmbim
+  SECTION:=libs
+  CATEGORY:=Libraries
+  DEPENDS:=+glib2
+  TITLE:=Helper library and utils to talk to MBIM enabled modems
+  URL:=https://www.freedesktop.org/wiki/Software/libmbim
+endef
+
+define Package/libmbim/description
+  Helper library to talk to MBIM enabled modems.
+  Add mbim-utils for extra utilities.
+endef
+
+define Package/mbim-utils
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libmbim
+  TITLE:=Utilities to talk to MBIM enabled modems
+  URL:=https://www.freedesktop.org/wiki/Software/libmbim
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include
+	$(CP) \
+		$(PKG_INSTALL_DIR)/usr/include/libmbim-glib \
+		$(1)/usr/include/
+
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) \
+		$(PKG_INSTALL_DIR)/usr/lib/*.{so*,a,la} \
```

Comment:
Any reason why static libs are being built?

Reply:
Also, this might pull in more than intended, please make it more specific

Reply:
I removed the .a static library from here, but to be honest I could use a bit of guidance as to what really needs to be done.

Reply:
Based on your current branch, I created a copy and I inserted additional commits, see https://github.com/mhei/packages/tree/mm. I think this should address @diizzyy's comments. You could just squash my two commits into your original two commits, having still 3 commits finally. Please note, that I also re-ordered the commits because of dependency order: modemmanager -> libqmi -> libmbim. Adding modemmanager as first commit would break since it would depend on a library which is not (yet) available in the repo.

Reply:
Thanks for that! I hope I squashed it right